pandas/_libs/properties.pyi

```diff
@@ -13,7 +13,7 @@ from pandas._typing import (
 # note: this is a lie to make type checkers happy (they special
 # case property). cache_readonly uses attribute names similar to
 # property (fget) but it does not provide fset and fdel.
-cache_readonly = property
+cache_readonly = ...
```

Comment:
I hope this causes mypy&pyright errors because it should :) Might need an ignore if `cache_readonly: TypeAlias = property` doesn't work

Reply:
There are errors about this: https://github.com/pandas-dev/pandas/actions/runs/4620671070/jobs/8171100725?pr=52452